web/src/repo/LSIFVerification.tsx

```diff
@@ -68,62 +89,43 @@ export class LSIFVerification extends React.PureComponent<Props, State> {
         return (
             <div className="lsif-verification">
                 {this.state.tokenOrError && !isErrorLike(this.state.tokenOrError) ? (
-                    <div className="alert alert-success">
-                        <CheckIcon className="icon-inline" /> Topic found. Here's the LSIF upload token:
+                    <div className="alert alert-success mb-1">
+                        <CheckIcon className="icon-inline mb-1" /> Topic found. Here's the LSIF upload token:
```

Comment:
This is an inline element, so a margin shouldn't do anything on it
```suggestion
                        <CheckIcon className="icon-inline" /> Topic found. Here's the LSIF upload token:
```